orchestrator/gravity_utils/src/connection_prep.rs

```diff
@@ -236,7 +236,7 @@ pub async fn check_delegate_addresses(
     prefix: &str,
 ) {
     let eth_response = client
-        .get_delegate_key_by_eth(QueryDelegateKeysByEthAddress {
+        .delegate_keys_by_validator(DelegateKeysByValidatorAddress {
```

Comment:
@jackzampolin I wasn't 100% sure about this rename; mind confirming I picked correctly on this?